src/amo/components/SearchPage.js

```diff
@@ -45,9 +46,12 @@ export default class SearchPage extends React.Component {
 
     return (
       <div className="SearchPage">
+        {hasSearchParams ? (
```

Comment:
I think I meant "showing stuff" and not "adding stuff". The happy path doesn't get users to a blank search page but if they went there and there are no results I wouldn't want a sort. And also if they search and get "no results" a search sort is weird and also useless. The entire page changes on search and based on whether there are results or not (the paginator does the same thing), so I think conditionally showing/hiding search sort makes sense.